modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/pagination/Page.java

```diff
@@ -70,6 +70,26 @@ public static <T> Page<T> of(
 		return new Page<>(actions, facets, items, pagination, totalCount);
 	}
 
+	public Page(
```

Comment:
Instead of public can we make it protected to ensure only subclasses can call it?